about.html

```diff
@@ -7,65 +7,110 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/about.css">
```

Comment:
You didn't need to create a separate page for the about css but good work still!